src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -118,7 +110,7 @@ export const SecurityAnalysisParameters: FunctionComponent<{
     const toLimitReductions = useCallback(
         (formLimits: Record<string, any>[]) => {
             if (!params?.limitReductions) {
-                return null;
+                return [];
             }
             return params?.limitReductions.map((vlLimits: ILimitReductionsByVoltageLevel, indexVl: number) => {
```

Comment:
no need of ? check in params.limitReductions

Reply:
params is of type "ISAParameters | null", I think it's needed

Reply:
fixed